mappings/net/minecraft/block/sign/SignType.mapping

```diff
@@ -5,6 +5,7 @@ CLASS net/minecraft/unmapped/C_xlaykyai net/minecraft/block/sign/SignType
 	FIELD f_khfupynz DARK_OAK Lnet/minecraft/unmapped/C_xlaykyai;
 	FIELD f_lxkmsfqj MANGROVE Lnet/minecraft/unmapped/C_xlaykyai;
 	FIELD f_nldrovwk WARPED Lnet/minecraft/unmapped/C_xlaykyai;
+	FIELD f_npkrgudo SIGN_TYPE_CODEC Ljava/util/Map;
```

Comment:
```suggestion
	FIELD f_npkrgudo CODEC Ljava/util/Map;
```

Reply:
There's already an autogenerated `CODEC` field right below this, and it's not a Codec anyways, so changed it to be called `TYPES`, since it's a map of all the sign types.